load.php

```diff
@@ -137,6 +137,34 @@ function( $module_settings ) {
 	return $modules;
 }
 
+/**
+ * Gets the active and valid performance modules.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $module Slug of the module.
+ * @return null|string Module slug name on success, null on failure.
```

Comment:
This is a bit odd of a return value, I think this should return a boolean (`false` or `true`) since it is an "is ..." check. Since the developer passes `$module` into the function anyway, no point to return that same value.

Reply:
Thanks for the feedback. The PR was updated to reflect these changes.